doc/api/process.md

```diff
@@ -1664,6 +1683,7 @@ if (process.getuid && process.setuid) {
 
 This function is only available on POSIX platforms (i.e. not Windows or
 Android).
+This feature is not available in [`Worker`][] threads.
```

Comment:
I think this was intended by me, because the sentence before also explains when this feature is not available → it made sense to me to have them be a single paragraph

Reply:
Maybe it is worth to join the same sentences in these sections: `process.initgroups()`, `process.setegid()`, `process.seteuid()`, `process.setgid()`, `process.setgroups()`. Feel free to ignore, it is not so important in such heavy PR)

Reply:
@vsemozhetbyt Yup, done! (Feel free to push documentation change commits to this branch, btw.)